android/app/build.gradle

```diff
@@ -111,20 +136,17 @@ android {
             signingConfig signingConfigs.debug
         }
         release {
-            signingConfig signingConfigs.release
+            signingConfig signingConfigs.debug
+            productFlavors.production.signingConfig signingConfigs.release
```

Comment:
Hey @kowczarz, do you recall why line 140 is necessary? Is this doing something that the following code would not? 

```
signingConfig signingConfigs.release
```